lmms_eval/models/internvl2.py

```diff
@@ -80,7 +80,7 @@ def dynamic_preprocess(image, min_num=1, max_num=6, image_size=448, use_thumbnai
     return processed_images
 
 
-def load_image(image, input_size=448, max_num=6):
+def load_image(image, input_size=448, max_num=25):
```

Comment:
Kindly pin @pufanyi to help check whether the internvl2 should use this settings. Thank you!

Reply:
Hi, it seems that in the official code of `internvl2`, they use `max_num=6` as default:

https://github.com/OpenGVLab/InternVL/blob/c7c5af1a8930b4862afe8ed14672307082ef61fa/internvl_chat/internvl/train/dataset.py#L702

If you really want to change it, maybe we can change our code to add this parameter to `generation_kwargs`.

Reply:
ic, will change it to the default setting